backend/curriculum_tracking/tests/test_RecruitProject.py

```diff
@@ -82,3 +84,23 @@ def test_too_long(self):
         self.assertIn("-test-project-", repo_name)
         self.assertIn(self.user.first_name, repo_name)
         self.assertIn(self.user.last_name, repo_name)
+
+
+class get_total_duration(TestCase):
+    def test_returns_correct_duration(self):
+        card = factories.AgileCardFactory(
+            content_item=factories.ContentItemFactory(
+                content_type=models.ContentItem.PROJECT, project_submission_type="L"
+            ),
+            status=models.AgileCard.READY,
+        )
+        card.recruit_project.start_time = datetime(
+            2024, 2, 12, 14, 6, 17, 373514, tzinfo=timezone.utc
+        )
+        card.recruit_project.end_time = datetime(
+            2024, 2, 12, 15, 6, 17, 373514, tzinfo=timezone.utc
+        )
+
+        self.assertEquals(
+            card.recruit_project.get_total_duration, timedelta(seconds=3600)
+        )
```

Comment:
What happens if the end_time is empty?
What happens if the start and end times are empty?

Consider edge cases.